api/include/opentelemetry/trace/noop.h

```diff
@@ -47,12 +48,13 @@ class NoopSpan final : public Span
 
   bool IsRecording() const noexcept override { return false; }
 
-  Tracer &tracer() const noexcept override { return *tracer_; }
+  SpanContext GetContext() const noexcept override { return span_context_; }
 
   void SetToken(nostd::unique_ptr<context::Token> && /* token */) noexcept override {}
 
 private:
   std::shared_ptr<Tracer> tracer_;
+  SpanContext span_context_;
```

Comment:
how is this being set ? should it be in constructor ?

Reply:
This is just a placeholder to conform to its parent class Span, I don't think it's being used.